pkg/dev/dev.go

```diff
@@ -53,8 +58,54 @@ func (o *DevClient) Start(devfileObj parser.DevfileObj, platformContext kubernet
 	if err != nil {
 		return err
 	}
-	klog.V(4).Infoln("Successfully created inner-loop resourcs")
-	fmt.Fprintf(out, "\nYour application is running on cluster.\n ")
+	klog.V(4).Infoln("Successfully created inner-loop resources")
+
+	return nil
+}
+
+func (o *DevClient) Cleanup() error {
+	var err error
+	return err
+}
+
+func (o *DevClient) SetupPortForwarding(portPairs []string, devfileObj parser.DevfileObj, errOut io.Writer) error {
+	pod, err := o.kubernetesClient.GetOnePodFromSelector(componentlabels.GetSelector(devfileObj.GetMetadataName(), "app"))
```

Comment:
Couls we pass the "app" constant from the CLI, to make the business layer able to work with different apps?

Reply:
Or, instead, you could pass the pod as a parameter, so this method would be much more generic (it would forward ports for a specific pod, without dealing with apps, devfiles, etc) and could be placed in the `kclient` package.

Reply:
Are you suggesting to find the matching pod in the `pkg/odo/cli/dev/dev.go` instead?